### PR TITLE
CVSL-1370 add test and some renaming

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -386,7 +386,7 @@ module.exports = function createApp({
     secureRoute(licenceSearchRouter(licenceSearchService), { auditKey: 'LICENCE_SEARCH' })
   )
   app.use(
-    '/admin/licencesWithCOM/',
+    '/admin/downloadLicencesWithCOM/',
```
Comment (louisenorris) [inline server/app.js:389]: Think this is a little better 😅

File: server/services/licenceSearchService.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ module.exports = function createLicenceSearchService(
         { id: 'prisonerNumber', title: 'PRISON_NUMBER' },
         { id: 'prisonId', title: 'PRISON_ID' },
         { id: 'transitionDate', title: 'TRANSITION_DATE' },
```
Comment (andrewrlee) [inline server/services/licenceSearchService.js:64]: maybe call this hand over date?

File: server/services/licenceSearchService.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ module.exports = function createLicenceSearchService(
         { id: 'prisonerNumber', title: 'PRISON_NUMBER' },
         { id: 'prisonId', title: 'PRISON_ID' },
```
[the remark this answers is not in the document]
Comment (louisenorris) [reply server/services/licenceSearchService.js:64]: Easily added so included since latest commit 😄

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**